distro.py

```diff
@@ -1074,7 +1074,72 @@ def _parse_distro_release_content(line):
         return distro_info
 
 
-_distro = LinuxDistribution()
+def _normalize_id(distro_id, table):
+    """ Helper function which normalizes a distro id value. """
+    distro_id = distro_id.lower().replace(' ', '_')
+    return table.get(distro_id, distro_id)
+
+
+class LinuxMintDistribution(LinuxDistribution):
+    """ LinuxMint will have the same /etc/os-release file
+    as it's upstream Ubuntu so we should not use any info
+    from that file except `ID_LIKE`. """
+    def os_release_attr(self, attribute):
+        if attribute == 'id_like':
+            v = super(LinuxMintDistribution, self).os_release_attr(attribute)
+            return v
+        return ''
+
+
+class CloudLinuxDistribution(LinuxDistribution):
+    """ CloudLinux <7 doesn't provide an lsb-release or os-release file,
+    only a redhat-release file so it's id is incorrectly detected as `rhel`.
+    """
+    def id(self):
+        return 'cloudlinux'
+
+
+class DebianDistribution(LinuxDistribution):
```

Comment:
Okay, I'll add a `DebianStretchDistribution` class. :) I'm guessing because Linux Mint has the os-release problem it'll stay with a single class until Linux Mint fixes that issue?